app/assets/stylesheets/forever_style_guide/modules/_nav-icons.scss

```diff
@@ -1,63 +1,70 @@
-$nav-icon-width: 60px;
 $nav-icon-margin: 3px;
+$cart-count-xsmall-offset: 2px;
+$cart-count-mobile-offset: 6px;
 $cart-count-width: 20px;
+$cart-icon-xsmall-width: 30px + 10px; //10px to account for extra spacing
+$icon-width: 28px;
+$cart-tab-padding: 20px;
+$cart-icon-vertical-offset: 7px;
+$cart-icon-vertical-offset-mobile: 4px;
+$deals-icon-horizontal-offset: 3px;
+$deals-icon-horizontal-offset-mobile: 4px;
+$chevron-bottom-alignment-offset: 15px;
+$chevron-right-alignment-offset: 5px;
+$screen-below-xs: $screen-xs - 1;
 
 @mixin chevron-style {
-  position: absolute;
   right: 15px;
   font-size: $font-size-default;
 }
 
 .nav-icon-stack {
-  width: $nav-icon-width;
-  padding: 7px 4px 4px 4px !important; // overriding .navbar-nav > li > a
+  padding: 3px 4px 4px 4px !important; // overriding .navbar-nav > li > a
   text-align: center;
 
   .fa {
     display: block;
     position: relative;
     top: $nav-icon-margin;
   }
-}
-.nav-icon-label {
-  font-size: $font-size-100;
-  line-height: $font-size-400;
 
-  &.has_count {
-    position: absolute;
-    top: 26px; // align labels
-  }
-}
-
-// Style offcanvas account icons consistent w/ product dropdowns
-@media (max-width: $grid-float-breakpoint-max) {
-  .navbar-account_item .nav-icon-stack {
-    width: auto;
-    padding: 15px !important; // override .navbar .nav.navbar-nav > li > a
-    text-align: left;
+  .icon-bar {
+    display: block;
+    width: $icon-width;
+    height: 4px;
+    border-radius: 4px;
+    margin: 3px auto 0 auto;
```

Comment:
The 3px margin-top here isn't having any effect b/c the subsequent &:first-of-type pseudo-class and then there's a default bootstrap style for  ```.navbar-toggle .icon-bar + .icon-bar```